owl/EASE-OBJ.owl

```diff
@@ -2652,6 +2681,20 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#StructuralDesign -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#StructuralDesign">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design"/>
+        <rdfs:comment>Structural design is the methodical investigation of the stability, strength and rigidity of structures and considers the way in which parts of a system or object are arranged. 
```

Comment:
how is a design an "investigation"? An investigation seems rather to be a Situation to me.

Reply:
how about `Structural design is the description of`?